docker/ha/init.sql

```diff
@@ -1,24 +1,24 @@
-CREATE USER root;
+CREATE USER orion;
```

Comment:
@lucassaldanha @MadelineMurray is this something we need to call attention to in the db?